### PR TITLE
Adm 253: Increase the test coverage for utils & steps-fetch & token-verify service file in frontend

### PR DESCRIPTION
**Pull request by Huanran-Cloud**
https://dorametrics.atlassian.net/jira/software/projects/ADM/boards/2?selectedIssue=ADM-253

### CODE REVIEW

File: frontend/src/app/dora/service/token-verify.service.spec.ts [hunk 1]
```diff
@@ -1,16 +1,33 @@
...
+import { async } from '@angular/core/testing';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
 
 import { TokenVerifyService } from './token-verify.service';
 
 describe('TokenVerifyService', () => {
-  let service: TokenVerifyService;
+  const tokenVerifyService = new TokenVerifyService();
+  let formGroup = new FormGroup({ first: new FormControl('first name') });
+  let formGroupBig = new FormGroup({ first: new FormControl('first name') });
```
Comment (xxj0528) [inline frontend/src/app/dora/service/token-verify.service.spec.ts:9]: what means `formGroupBig`

File: frontend/src/app/dora/components/config-stepper/configuration/configuration.component.spec.ts [hunk 2]
```diff
@@ -38,4 +38,13 @@ describe('ConfigurationComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('check initial from values for config form group', () => {
```
Comment (chaoyangyes) [inline frontend/src/app/dora/components/config-stepper/configuration/configuration.component.spec.ts:42]: maybe you can use 'should' in test case name and what is 'initial from values"?

**Review by chaoyangyes — APPROVED** [review]
LGTM

**Review by xxj0528 — APPROVED** [review]
LGTM

**Pull request merged**